apps/src/sites/studio/pages/code-studio.js

```diff
@@ -44,6 +44,7 @@ window.dashboard.videos = require('@cdo/apps/code-studio/videos');
 window.dashboard.assets = require('@cdo/apps/code-studio/assets');
 window.dashboard.pairing = require('@cdo/apps/code-studio/pairing');
 window.dashboard.teacher = require('@cdo/apps/code-studio/teacher');
+window.dashboard.project = require('@cdo/apps/code-studio/initApp/project');
```

Comment:
I have an idea for cleaning up these global references:

```jsx
Object.assign(window.dashboard, {
  get project() {
    if (process.env.NODE_ENV !== 'production') {
      console.error("You should import project directly from '@cdo/apps/code-studio/initApp/project'");
    }
    return require('@cdo/apps/code-studio/initApp/project');
  }
});
```

What do you think about that?